app/src/test/java/com/example/android/architecture/blueprints/todoapp/statistics/StatisticsViewModelTest.kt

```diff
@@ -100,9 +103,9 @@ class StatisticsViewModelTest {
     }
 
     @Test
-    fun loadTasks_loading() {
-        // Pause dispatcher so we can verify initial values
-        mainCoroutineRule.pauseDispatcher()
+    fun loadTasks_loading() = runTest {
+        // Override eagerly executing Main dispatcher for just a single test
+        Dispatchers.setMain(StandardTestDispatcher())
```

Comment:
Same here, should you pass the `StandardTestDispatcher ` to the `MainCoroutineRule` and then call `advanceUntilIdle` as well in the test method above?

Reply:
If you decide to keep this `setMain` here, maybe improve the comment? explain what `StandardTestDispatcher` does instead of the behavior you're overriding